src/test/scala/latis/input/TestHapiBinaryAdapter.scala

```diff
@@ -0,0 +1,47 @@
+package latis.input
+
+import java.net.URI
+
+import latis.data.{Data, Sample, SeqFunction}
+import latis.metadata.Metadata
+import latis.model._
+import latis.output.TextWriter
+import latis.util.FileUtils
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers._
+
+class TestHapiBinaryAdapter extends FlatSpec {
```

Comment:
I've been seeing people in other PRs appending "Spec" to their test classes instead of prepending them with "Test". Should this file/class be renamed to `HapiBinaryAdapterSpec`?

Reply:
Yeah, the idiom for ScalaTest is to end the name with `Spec`.

Reply:
I updated this test file and went ahead and did "HapiJsonAdapterSpec" too.